travel/src/main/java/com/tx/travel/controller/DailyAllowanceController.java

```diff
@@ -0,0 +1,97 @@
+package com.tx.travel.controller;
+
+import com.tx.travel.payload.request.DailyAllowanceRequest;
+import com.tx.travel.payload.response.DailyAllowanceResponse;
+import com.tx.travel.payload.response.MessageResponse;
+import com.tx.travel.service.DailyAllowanceService;
+import com.tx.travel.service.exception.DailyAllowanceNotFoundException;
+import com.tx.travel.service.exception.RegionAlreadyExistsException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.List;
+
+@CrossOrigin(origins = "*", maxAge = 3600)
+@RestController
+@RequestMapping("/api/daily-allowances")
```

Comment:
since /api is common prefix for all endpoints, you should set it on application level, check this https://www.baeldung.com/spring-boot-controllers-add-prefix#2-application-properties